CURL/build.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
```

Comment:
From where does this file come?
I don't see it in client's repo...

Reply:
I see you found it: https://github.com/NetsoftHoldings/hubstaff-client/pull/2027/commits/7206b00bf70abd5829baeba7d7a13b2999bbc5bc#r1832844602